modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/ScheduleContainer.tsx

```diff
@@ -31,15 +35,20 @@ export default function ScheduleContainer({
 	const [displayedScheduleValues, setDisplayedScheduleValues] = useState<{
 		[key in SchedulePropertyKey]: SchedulePropertyValues;
 	}>({
+		expirationDate: {
+			checked: scheduleProperties.expirationDate?.checked ?? true,
+			value: scheduleProperties.expirationDate?.value ?? '',
+		},
 		reviewDate: {
-			checked: scheduleProperties.reviewDate.checked,
-			value: scheduleProperties.reviewDate.value ?? '',
+			checked: scheduleProperties.reviewDate?.checked ?? true,
+			value: scheduleProperties.reviewDate?.value ?? '',
```

Comment:
Since the `reviewDate` and `expirationDate` are mandatory, we can revert these changes.